languages/definitions/Arabic.yml

```diff
@@ -0,0 +1,16 @@
+---
+locale: ar-JO
+dictionaryFile: ar-utf8.csv
+abcString: أﺏﺕ
+hasUpperCase: no
+layout:
+  - [SPECIAL] # 0
+  - [PUNCTUATION_AR] # 1
+  - [ب,ت,ة,ث] # 2
```

Comment:
The letters probably need to be listed backwards, so that they would appear correctly on the screen. Must double check this before merging.

Reply:
i have done some testing with this exact file in the phone and it types properly with the letters listed like this, keep in mind that if you do not have arabic locale/keyboard installed on your computer, depending on your text editor it could show in the reverse order, but the way that it is arranged now is working and good.

Reply:
![image](https://github.com/sspanak/tt9/assets/156047982/f1848d45-8ec4-4ef3-8099-bdda03464511)
![image](https://github.com/sspanak/tt9/assets/156047982/cdfd2ff7-2703-4625-ba1c-212c116f144a)

Even the online file editor in github is inconsistent with showing the arabic from right to left or left to right (the second image with the file viewer in edit mode is correct) but this is the format that works correctly to have accurate input on the phone.